mobile-save-for-later/src/main/scala/com/gu/sfl/controller/SaveArticlesController.scala

```diff
@@ -18,7 +18,16 @@ class SaveArticlesController(updateSavedArticles: UpdateSavedArticles)(implicit
   override def apply(lambdaRequest: LambdaRequest): Future[LambdaResponse] = {
     val futureResponse = lambdaRequest match {
       case LambdaRequest(Some(json),  _) =>
-        futureSave(Try.apply(mapper.readValue[SavedArticles](json)), lambdaRequest.headers)
+        val triedSavedArticles = Try.apply(mapper.readValue[SavedArticles](json))
+        triedSavedArticles match {
+          case Failure(t) => {
+            val headersWithoutAuth = lambdaRequest.headers.filterNot{ case (k,v) => k.toLowerCase.equals("authorization")}
```

Comment:
https://github.com/guardian/mobile-save-for-later/blob/master/mobile-save-for-later/src/main/scala/com/gu/sfl/lambda/LambdaApiGateway.scala#L47